cli/main.c

```diff
@@ -1143,35 +1143,38 @@ static int print_fw_part_info(struct switchtec_dev *dev)
 	int nr_mult = 16;
 	struct switchtec_fw_image_info act_img, inact_img, act_cfg, inact_cfg,
 		mult_cfg[nr_mult];
-	struct switchtec_fw_footer map;
-	struct switchtec_fw_footer bootloader;
-	char map_ver[16];
+	struct switchtec_fw_image_info bl2_act_img, bl2_inact_img;
+	struct fwdl_meta bootloader;
 	char bootloader_ver[16];
 	int bootloader_ro;
+	unsigned long partition_start;
+	size_t partition_length;
 	int ret, i;
 
+	/*fetch fw image info from GAS */
```

Comment:
Generally, we do not use comment like this, if a variable or a function name is clear enough, reader can understand easily.

Reply:
just remove comments.